README.md

```diff
@@ -1 +1,2 @@
 # flask_tutorial
+![](static/images/logo.jpg)
```

Comment:
you can use basic html and css in the md file if you want to customize the readme add alittle bit of general description and technology stack used in the project.

Reply:
np ill do it